data/json/items/tool/misc.json

```diff
@@ -785,5 +785,22 @@
       { "type": "firestarter", "moves": 1000, "moves_slow": 25000, "need_sunlight": true }
     ],
     "flags": [ "FIRESTARTER", "THERMOMETER" ]
+  },
+  {
+    "id": "aluminum_stepladder",
+    "type": "TOOL",
+    "name": { "str": "aluminum stepladder" },
+    "description": "This is an aluminum stepladder.  Use it to set it down.",
+    "looks_like": "stepladder",
+    "weight": "91500 g",
```

Comment:
I think an aluminum stepladder should be lighter than a wooden one.

https://www.laddersandscaffoldtowers.co.uk/blog/ladders/wooden-ladder-vs-aluminium-ladders/

Reply:
Not gonna lie, I thought so as well. This mass is the result of me doing proportion calculations between the volume-mass ratio of planks and the wooden stepladder to calculate what the aluminum one should be but either the method is incredibly flawed or I messed up the maths. I'll go with the average weight of a 6-foot aluminum stepladder here